neqo-transport/src/tracking.rs

```diff
@@ -397,9 +418,15 @@ impl RecvdPackets {
             unacknowledged_count: 0,
             unacknowledged_tolerance: DEFAULT_ACK_PACKET_TOLERANCE,
             ignore_order: false,
+            ecn_count: EcnCount::default(),
         }
     }
 
+    /// Increase the ECN count for the mark given by `ecn` by `n`.
+    pub fn inc_ecn_count(&mut self, ecn: IpTosEcn, n: u64) {
+        self.ecn_count[ecn] += n;
+    }
```

Comment:
Nit, given that `inc_ecn_count` is only ever called with `n = 1`:

```suggestion
    /// Increase the ECN count for the mark given by `ecn` by `1`.
    pub fn inc_ecn_count(&mut self, ecn: IpTosEcn) {
        self.ecn_count[ecn] += 1;
    }
```

Reply:
I had that originally, but was wondering if with multi-packet I/O we might want to increase by more than one. But I guess we can make that change at that point.